src/mock/data.js

```diff
@@ -16,19 +16,19 @@ export const heroData = {
 export const aboutData = {
   img: 'profile.jpg',
   paragraphOne:
-    'I’m Josephine Noble, but you can call me Jojo. I’m a Salesforce Administrator and Business Analyst living in Denver, Colorado. I have a degree in Humanities with a focus on Media Studies from the University of Colorado -  Boulder. I currently work for the Leeds School of Business at my alma mater. Go Buffs!',
+    'I’m Josephine Noble, but you can call me Jojo. I’m a Salesforce Architect/Admin living in Missoula, Montana. I have a BA degree in Humanities with a focus on Film Studies from the University of Colorado -  Boulder. I currently work remotely for Slalom.',
   paragraphTwo:
-    'I am an experienced problem solver. At my current job you can find me leading new project initiatives and campaigns for the different schools, tracking down project requirements from Assistant Deans, and also providing administrative and technical support for a wide variety of users at CU.',
+    'I am an experienced problem solver. At my current job you can find me leading new project initiatives and campaigns for different clients and industries, tracking down project requirements, and also providing both solutions and technical architectural support on a variety of apps.',
```

Comment:
apps? maybe 'applications' or even better 'services'

Reply:
Oh I like that.